src/deployment/3_deployContracts.js

```diff
@@ -403,36 +410,13 @@ async function main() {
         console.log('Actual nonce for globalExitRoot deployment', polygonZkEVMGlobalExitRoot.deployTransaction.nonce);
 
         try {
-            /*
-             * Initialization:
-             * Unfortunately, normal initilziation would throw an error: Function initialize not found
-             * await polygonZkEVMGlobalExitRoot.initialize(
-             *     forkingManagerContract.address,
-             *     parentContract,
-             *     precalculateZkevmAddress,
-             *     proxyBridgeAddress,
-             * );
-             * Hence we go with the custom initialization
-             */
-            const iface = new ethers.utils.Interface([
-                'function initialize(address,address,address,address)',
-            ]);
-            const dataCallInitialize = iface.encodeFunctionData(
-                'initialize(address,address,address,address)',
-                [
-                    forkingManagerContract.address,
-                    parentContract,
-                    precalculateZkevmAddress,
-                    proxyBridgeAddress,
-                ],
+            const iForkableExitRoot = await ethers.getContractAt('IForkableGlobalExitRoot', polygonZkEVMGlobalExitRoot.address);
```

Comment:
This is a much more consice formulation that is also easier to refactor, than writing the abi function down into code, as we did before